plugins/woocommerce-beta-tester/src/remote-logging/register-exception-filter.tsx

```diff
@@ -12,7 +12,7 @@ import { dispatch } from '@wordpress/data';
 /**
  * Internal dependencies
  */
-import { API_NAMESPACE } from './';
+import { API_NAMESPACE } from './constants';
```

Comment:
I moved the `API_NAMESPACE` to a constant file, to avoid all of the `remote-logging` to be bundled into this filter as well.
Given some of it is only needed in the settings page.